website/docs/advanced/notifications-webhooks.md

```diff
@@ -47,7 +51,301 @@ The structure of the JSON array injected into the **##ChangeDetails##** looks li
+
+Each webhook request contains the following headers:
+
+| Header                             | Description                                                                                         |
+| ---------------------------------- | --------------------------------------------------------------------------------------------------- |
+| `X-ConfigCat-Webhook-ID`           | The webhook request's unique identifier.                                                            |
+| `X-ConfigCat-Webhook-Timestamp`    | The time the webhook was sent in unix timestamp format. (seconds since epoch)                       |
+| `X-ConfigCat-Webhook-Signature-V1` | The list of the `base64` encoded HMAC-SHA-256 signatures. (comma delimited, 1 for each signing key) |
+
+Currently the latest (and the only) signature header version is `V1`. If the signing process is going to be changed in the future, more headers will be added with incremented version postfix.
+
+Example request:
+
+```
+POST /path HTTP/1.1
+Host: <your-host>
+X-ConfigCat-Webhook-ID: b616ca659d154a5fb907dd8475792eeb
+X-ConfigCat-Webhook-Timestamp: 1669580020
+X-ConfigCat-Webhook-Signature-V1: RoO/UMvSRqzJ0OolMMuhHBbM8/Vjn+nTh+SKyLcQf0M=,heIrGPw6aylAZEX6xmSLrxIWVif5injeBCxWQ+0+b2U=
+```
+
+### Content to sign
+The signature is calculated from the content constructed by concatenating the webhook's `id`, `timestamp`, and raw `body` together.
+
+```js
+const contentToSign = `${webhookId}${timestamp}${body}`
+```
+
+| Content part            | Description                                                                                                       |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `webhookId`             | The webhook's identifier received in the `X-ConfigCat-Webhook-ID` request header.                                 |
+| `timestamp`             | The timestamp value received in the `X-ConfigCat-Webhook-Timestamp` request header.                                       |
+| `body`                  | The raw body of the received webhook request. If the webhook doesn't have a request body it can be left out.      |
+
+:::caution
+The signature calculation is sensitive to any changes on the signed content, so it's important to not change the request body before the verification. 
+Otherwise, the calculated signature could be completely different from the value received in the  
+`X-ConfigCat-Webhook-Signature-V1` header.
+:::
+
+### Calculating signature
+ConfigCat uses `HMAC` with `SHA-256` to sign webhooks. You can retrieve the **signing key(s)** required for the signature calculation from the <a href="https://app.configcat.com/product/webhooks" target="_blank">ConfigCat Dashboard's webhook section</a>.
+
+<img src="/docs/assets/whsk.png" className="zoomable bottom-spaced" alt="signing keys" />  
```

Comment:
The censored part should be black or white, only the emphasized part should be red.